typebackend/serializers.py

```diff
@@ -27,7 +27,7 @@ def create(self,validated_data):
 
     def to_representation(self,instance):
         response=super(PractiseLogSerializer,self).to_representation(instance)
-        response['para']=Paragraph.objects.get(id=instance.para.id).para
+        response.pop('para')
```

Comment:
Maybe try [extra kwargs](https://www.django-rest-framework.org/api-guide/serializers/#additional-keyword-arguments) instead of this?

Reply:
Is it better ?

Reply:
yeah doesn't require this to_representation function